### PR TITLE
Add pre-conversion check in Convert2RHEL

### PR DESCRIPTION
**Pull request by Lennonka**
Documents https://github.com/theforeman/foreman_remote_execution/pull/840, i.e. Convert2RHEL analyze template

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.9 if available

### CODE REVIEW

**Comment by Lennonka** [comment]
@stejskalleos In which Foreman can this be expected? 3.9+?

**Comment by Lennonka** [comment]
This should be merged after #2604

File: guides/common/modules/con_converting-a-host-to-rhel.adoc [hunk 1]
```diff
@@ -62,6 +62,23 @@ endif::[]
...
+ifdef::managing-hosts[]
+For more information, see xref:executing-a-remote-job_{context}[].
+endif::[]
+ifndef::managing-hosts[]
+For more information, see {ManagingHostsDocURL}executing-a-remote-job_managing-hosts[Executing a Remote Job] in _{ManagingHostsDocTitle}_.
+endif::[]
+
++
+Review pre-conversion analysis reports and resolve any issues that prevent the conversion.
+Repeat this step until you resolve all issues.
```
Comment (stejskalleos) [inline guides/common/modules/con_converting-a-host-to-rhel.adoc:80]: I would add `resolve all issues that are blocking the conversion`

Some of the issues are not always problems users have to solve.

**Comment by stejskalleos** [comment]
In which Foreman can this be expected? 3.9+? Yes
Is this step optional? Well, yes and know. Yes if you are an adventurous person, otherwise no. You can start the conversion process without it, but that doesn't sound to me like a good idea.
Should this step replace any prerequisites? I don't think so.

**Review by adamlazik1 — APPROVED** [review]
Looks good stylewise.

**Comment by Lennonka** [comment]
I've verified that this has to be cherry-picked:
- f8376970c..77edff987  3.8 -> 3.8
- bbcd4e289..4001a5405  3.7 -> 3.7

**Pull request merged**